src/metabase/api/util.clj

```diff
@@ -16,5 +17,11 @@
   (check-superuser)
   (logger/get-messages))
 
+(defendpoint GET "/stats"
+  "Anonymous usage stats. Endpoint for testing, and eventually exposing this to instance admins to let them see
+  what is being phoned home."
+  []
+  (check-superuser)
+  (stats/get-anonymous-usage-stats))
```

Comment:
looks like you forgot to update this after renaming the fn to `anonymous-usage-stats`